### PR TITLE
Digipeater - Different LoRa Settings for RX and TX

### PR DESCRIPTION
**Pull request by dj1an**
Added Config Parameters and sendMessage code for Frequency / Mode Split Operation
All Parameters can now be set seperately for Rx and Tx.

This may be useful for Digipeater -> iGate Links.

### CODE REVIEW

File: src/project_configuration.h [hunk 1]
```diff
@@ -66,15 +66,18 @@ class Configuration {
...
     int  power;
-    int  spreadingFactor;
-    long signalBandwidth;
-    int  codingRate4;
+    int  spreadingFactorRx;
+    long signalBandwidthRx;
+    int  codingRate4Rx;
+    int  spreadingFactorTx;
+    long signalBandwidthTx;
+    int  codingRate4Tx;
```
Comment (peterus) [inline src/project_configuration.h:80]: use one naming convention in all files. please use Rx and Tx at the beginning in json-file, project_configuration.{c|cpp}, ...

like:

```
rxSpreadingFactor
rxSignalBandwidth
txSpreadingFactor
txSignalBandwidth
```

**Review by peterus — CHANGES_REQUESTED** [review]

File: src/project_configuration.h (hunk 1 above)
Comment (dj1an) [reply src/project_configuration.h:80]: yeah.. i just took the convention you used ;)
frequency_rx, frequency_tx / RxFrequency, TxFrequency ... also not very consistent

**Comment by dj1an** [comment]
Please check, fixed a lot of Variable names

**Review by peterus — APPROVED** [review]

**Comment by peterus** [comment]
@dj1an thanks for the changes!

**Pull request merged**